Makefile

```diff
@@ -44,19 +53,13 @@ bench:
 get-deps:
 	go get -t ./...
 
-ifndef GOBIN
-export GOBIN := $(GOPATH)/bin
-endif
-
 .PHONY: get-tools
 get-tools: GOPATH := $(shell mktemp -d)
 get-tools:
 	@echo "==> Installing tools at $(GOBIN)..."
 	@mkdir -p $(GOBIN)
 
-	@go get -u github.com/golang/lint/golint
 	@go get -u github.com/client9/misspell/cmd/misspell
-	@go get -u github.com/gordonklaus/ineffassign
-	@go get -u honnef.co/go/tools/cmd/megacheck
+	@go get -u github.com/golangci/golangci-lint/cmd/golangci-lint
```

Comment:
For CI I recommend installing binary version by curl to fix the version of linter

Reply:
Well spotted, is there a way to install the latest version automatically?

Reply:
Yes, but it has the same problem as go get-ing: one update of golangci-lint can cause failures of all your CI builds

Reply:
Done, I tried using the service but cannot access org public repos...